engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -1342,7 +1347,10 @@ namespace dmGraphics
         vk_present_info.pResults           = 0;
 
         res = vkQueuePresentKHR(context->m_LogicalDevice.m_PresentQueue, &vk_present_info);
-        CHECK_VK_ERROR(res);
+        if (res != VK_SUCCESS && res != VK_SUBOPTIMAL_KHR)
+        {
+            CHECK_VK_ERROR(res);
+        }
```

Comment:
This is a fix / workaround for android where the swap chain capabilities can have a presentation transform. For now we skip the transform completely by setting the currentTransform = identity, but that causes the presentation function to return a suboptimal result, which we don't care about right now.

Reply:
Would this be good info as a code comment, or is it easy to figure out?

Reply:
Yeah it's probably a good code comment, I will forget it otherwise